default.json5

```
@@ -3,10 +3,11 @@
   description: "This config defines the default Renovate rules that will be used in HiveMQ Product repos.",
   "extends": [
     "config:base",
-    "schedule:nonOfficeHours"
+    "schedule:earlyMondays"
```

Since the issue is regarding the automerging behavior, I would suggest to use the preset `schedule:automergeWeekends`. And since this is a T&E specific request, how about we create a separate renovate config for T&E that includes this change and you use that in your respective repos? I can help in making those changes. The dockerfile addition can ofc stay here.

---

I'd like to avoid a config fragmentation if possible. And I hope that we don't need different dependency management strategies for the additional 7 or 8 repositories of the broker teams.

Disclaimer: This is not meant as blame to any team. The renovate PRs on `hivemq-enterprise` are open since beginning of November. I don't think that a reduction of the schedule frequency will do any harm. It will effectively just reduce the amount of pending PRs and used Jenkins resources.

For example, if we keep the schedule on daily and auto-merge on weekly. Everything pending is auto-merged on Monday. Then we get a dependency update on Tuesday and another dependency is updated on Thursday. Then we have a PR pending for almost a week, before the next weekly auto-merge is running. And due to the updates during the week, we get two Jenkins runs on the PR (and another on `master` when it's merged and maybe on other PRs after rebasing). If we schedule and auto-merge once a week, it's less overhead in total.

I think the opposite settings make much more sense: Schedule once a week, but auto-merge daily (so the scheduled PRs are merged quickly after they are green). And the non-automerge PRs are created bundled, with all collected changes. So if we can quickly review and merge them on Mondays, we are done for the week without new PRs popping up and claiming focus.

Also if we will setup some kind of notification system (T&E will eventually need this, so we don't have to manually check our 40+ repositories for renovate PRs), we would get notifications from the schedule alone, not just from the merging. A reduction in frequency would also be good here.

---

I was by chance in the BFT daily, the also wouldn't mind the scheduling change to weekly.

---

You raise a fair point. I like the idea of scheduling once a week and auto-merging daily, especially if it reduces toil for you folks. 
Regarding the notifications, it shouldn't be hard to set up sth that pings you on your team slack channel. I will pitch sth on that area.

---

```suggestion
    "schedule:weekends"
```
With your suggestion, it will only have four hours to run through all our repos. Since we are relying on a hosted renovate service which doesn't have any SLAs, let's give it a bigger window to run those jobs.